crane-cloud/backend
# Ft admin disable projects

# Description

Allows admin or user to disable and enable projects

## Type of change

Please delete options that are not relevant.




## Trello Ticket ID

https://trello.com/c/09QfURga/1298-implement-project-management-disable-enable-by-admin-or-user-backend

## How Can This Be Tested?

Use end point /projects/{project_id}/disable [POST] to disable a project. Project is set to disabled and access to databases in project should be locked as well. Use end point /projects/{project_id}/enable [POST] to enable a project. Access to databases should be returned and project will be set back from being disabled.


## Checklist:
